### PR TITLE
[1.0] Remove array.prototype.find as not supported in IE fixes #974

### LINKED ISSUE #974: [1.0] App bundle not transpiled or minified
It looks like the result of `gatsby build` is not minified or transpiled fully. While testing IE11 on browserstack it can't load the script because there are still backtick characters.

If I try to load `https://gatsbygram.gatsbyjs.org/` it has the same issue.

The generated bundle contains this 

```
/***/ "./node_modules/gatsby-plugin-google-analytics/gatsby-browser.js":
/***/ (function(module, exports, __webpack_require__) {

	"use strict";
	
	exports.onRouteUpdate = function (location) {
	  // Don't track while developing.
	  if (("production") === `production`) {
	    ga(`set`, `page`, location.pathname);
	    ga(`send`, `pageview`);
	  }
	};

/***/ }),

```

Is there some setting I should tweak maybe? I would expect IE11 to be supported out of the box.

### PR DESCRIPTION
**Pull request by KyleAMathews**

### CODE REVIEW

File: packages/gatsby/src/cache-dir/production-app.js [hunk 1]
```diff
@@ -18,24 +18,40 @@ window.matchPath = matchPath
 import requires from "./async-requires"
 
 // Find page
-const findPage = pathname =>
-  pages.find(page => {
+const findPage = pathname => {
+  let foundPage
+  // Array.prototype.find is supported in IE so we use this
```
Comment (0x80) [inline packages/gatsby/src/cache-dir/production-app.js]: *not* supported

Comment (KyleAMathews) [inline packages/gatsby/src/cache-dir/production-app.js]: Oy, fixing.

Comment (0x80) [inline packages/gatsby/src/cache-dir/production-app.js]: why not use lodash.find?

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 15576772458ba5071e1dc79c5283a6e9096ced5c

https://app.netlify.com/sites/gatsbyjs/deploys/592402d9cf321c53095b1137

File: packages/gatsby/src/cache-dir/production-app.js [hunk 2]
```diff
@@ -18,24 +18,40 @@ window.matchPath = matchPath
...
+  // weird work around.
+  pages.some(page => {
```
Comment (0x80) [inline packages/gatsby/src/cache-dir/production-app.js]: why not use lodash.find?

**Comment by fabien0102** [comment]
mmm, this code is not compiled by babel? (babel should already use a polyfill for that, the lodash one maybe)

File: packages/gatsby/src/cache-dir/production-app.js (hunk 2 above)
Comment (KyleAMathews) [inline packages/gatsby/src/cache-dir/production-app.js]: Extra weight. Core should be as slim as possible so rely on browser primitives whenever possible.

**Comment by KyleAMathews** [comment]
@fabien0102 http://discuss.babeljs.io/t/why-does-array-prototype-find-not-work-with-babel-preset-es2015/46

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit d8051326c6bb28cd5d3d3de4b91a57544ade49c6

https://app.netlify.com/sites/gatsbyjs/deploys/5924077e6f4c5037ef3a5c65

**Pull request merged**